engine/gamesys/src/gamesys/test/test_gamesys.h

```diff
@@ -433,6 +434,7 @@ void GamesysTest<T>::SetUp()
     dmResource::RegisterTypes(m_Factory, &m_Contexts);
 
     dmGraphics::InstallAdapter();
+    dmGraphics::ResetDrawCount(); // for the unit test
```

Comment:
Since more tests use the rendering, we need to be able to reset the unit test flag.